static/app/components/searchQueryBuilder/types.tsx

```diff
@@ -25,3 +25,25 @@ export type CallbackSearchState = {
   parsedQuery: ParseResult | null;
   queryIsValid: boolean;
 };
+
+/**
+ * This is a list of operators that are used in the search query builder. That are only
+ * present on the frontend. This is because of the wildcard operators, and the fact that
+ * we utilize the underlying '*' character rather than introducing new operators on the
+ * backend.
+ */
+export enum ExtendedTermOperators {
+  DEFAULT = '',
+  GREATER_THAN_EQUAL = '>=',
+  LESS_THAN_EQUAL = '<=',
+  GREATER_THAN = '>',
+  LESS_THAN = '<',
+  EQUAL = '=',
+  NOT_EQUAL = '!=',
+  CONTAINS = 'contains',
+  DOES_NOT_CONTAIN = 'does not contain',
+  STARTS_WITH = 'starts with',
+  ENDS_WITH = 'ends with',
+}
+
+export type FETermOperators = ExtendedTermOperators | TermOperator;
```

Comment:
Shouldn't this be:

```tsx
enum WildcardOperators {
  CONTAINS = 'contains',
  DOES_NOT_CONTAIN = 'does not contain',
  STARTS_WITH = 'starts with',
  ENDS_WITH = 'ends with',
}

export type ExtendedTermOperators = TermOperator | WildcardOperators
```

That way there isn't any overlap

Reply:
Ohhhhhh i see what you're talking about now, yea that resolves the issues! Nice shout 😄